src/components/common/SceneWrapper.tsx

```diff
@@ -118,6 +125,13 @@ interface SceneWrapperProps {
 
   // True to make the scene scrolling (if avoidKeyboard is false):
   scroll?: boolean
+
+  // Optional KeyboardAccessoryView rendered as a sibling to the SceneWrapper content:
+  kavProps?: {
```

Comment:
I don't know if `kavProps` is an awesome name. I would call it `avoidKeyboard` or something like that.